doc/grpc_client/design_doc.md

```diff
@@ -0,0 +1,111 @@
+## gRPC client for active-actve DualToR scenario design
+
+## Revision
+
+| Rev | Date     | Author          | Change Description |
+|:---:|:--------:|:---------------:|--------------------|
+| 0.1 | 04/1/22 | Vaibhav Dahiya  | Initial version    |
+
+## Scope
+
+### Overview
+
+This document summarizes the approach taken to accommodate gRPC client
+for DualToR active active scenario. The gRPC client daemon's
+main purpose is to provide a way for linkmgr to exchange RPC's with SoC
+and do this within SONiC PMON docker
+
+
+Challenge: to provide an interface for gRPC daemon to exchange RPC's with the gRPC server running on the SoC
```

Comment:
Please follow the other templates and have this as part of "Requirement" section

Reply:
fixed